### ISSUE TITLE
Portrait mode

### ISSUE THREAD

**Issue by shockergit** [SkyTubeTeam/SkyTube#1016]
Please add portrait mode for official video mode

**Comment by gzsombor** [comment]
I don't think this is possible, if the 'offical Youtube' player does not support it.

**Comment by shockergit** [comment]
> I don't think this is possible, if the 'offical Youtube' player does not support it.

Official app supports it. App plays in portrait mode by default

**Comment by drogga** [comment]
@shockergit I don't get why would you want to use the YT embed player of the Extra build, which has limited functionality compared to the Default one...

**Comment by shockergit** [comment]
> @shockergit I don't get why would you want to use the YT embed player of the Extra build, which has limited functionality compared to the Default one...

Because i watch videos in portrait mode. When i start play in skytube it automatically goes to landscape full screen. Too irritating

**Comment by drogga** [comment]
THEN JUST USE THE "DEFAULT" PLAYER, instead of the "Official" one, which is the YouTube Embed player that is being called to play the video !, if you don't have the YouTube app installed, nothing will be played with this "player".

**Comment by drogga** [comment]
@shockergit
THEN JUST USE THE "DEFAULT" PLAYER, instead of the "Official" one, which is the YouTube Embed player that is being called to play the video !, if you don't have the YouTube app installed, nothing will be played with this "player".

Instructions to change it:
Open/launch/run SkyTube app, tap on the 3-dots to call the context menu, then on "Preferences">"Video Player">"Video Player">"Default" (1st option), it will play in whatever orientation you want.

**Comment by shockergit** [comment]
I don't want to install youtube app

**Comment by drogga** [comment]
@shockergit I don't want to offend you, but you are basically begging for that, LEARN ENGLISH so you can comprehend / understand what I wrote you.

@gzsombor
I think that you can close this (it's kind of invalid and incomplete), in case you have no interest in trying to force the YT's own embed player activity to respect the orientation.

**Comment by shockergit** [comment]
Because all youtube app dont play straight to full screen when starting a video

**Comment by shockergit** [comment]
Because all youtube app dont play straight to full screen when starting a video
